acceptance/bundle/deployment/bind/dashboard/databricks.yml

```diff
@@ -0,0 +1,12 @@
+bundle:
+  name: bind-dashboard-test-$BUNDLE_NAME_SUFFIX
+
+resources:
+  dashboards:
+    dashboard1:
+      display_name: $DASHBOARD_DISPLAY_NAME
+      warehouse_id: $TEST_DEFAULT_WAREHOUSE_ID
+      embed_credentials: true
+      serialized_dashboard: '{"pages":[{"name":"02724bf2","displayName":"Page 1"}]}'
```

Comment:
I think the dev flow we suggest for customers with dashboards is a bit different from other resources. https://docs.databricks.com/aws/en/dev-tools/bundles/resources#example-2

We generate the dashboards with file_path option and a file containing the dashboard instead of serialised dashboard
https://docs.databricks.com/aws/en/dev-tools/cli/bundle-commands#generate-dashboard-configuration
But if you attempt to deploy a bundle that contains a dashboard JSON file that is different than the one in the remote workspace, an error will occur. So I'm not interily sure if we should support bind, @pietern WDYT?

Reply:
But it's only because the content of the dashboard file is the same, right? If you change the content it will error out I guess?

Reply:
CLI will apply the change from file_path or serialized_dashboard unless there was a separate change done to the dashboard in the workspace. Becasue of that binding a dashboard might still be useful capability in some use cases